### PR TITLE
add markdown generator functionality

### PR DESCRIPTION
**Pull request by MichaelMai2000**
edit readme
add markdown sample

### CODE REVIEW

**Comment by MichaelMai2000** [comment]
More features: (Maybe)

1. Skip months in which no anime was finished.
2. Add corresponding images in Markdown.
3. Add links to corresponding Bangumi pages.

File: bangumi_report.py [hunk 1]
```diff
@@ -177,23 +177,67 @@ def __init__(self, image_url_list, user_id, year, type):
...
+        html = template.render(
+            user_id=self.user_id, image_list=self.image_url_list, year=self.year, type=self.type)
         if not to_stdout:
-            with open(file_name, 'w', encoding='utf-8') as f:
+            with open(self.file_name, 'w', encoding='utf-8') as f:
                 f.write(html)
         else:
             print(html)
 
+    def generate_markdown(self):
```
Comment (Shell32-Natsu) [inline bangumi_report.py]: Please make this function the same as `generate_html` to make it be able to print to stdout.

Comment (MichaelMai2000) [reply bangumi_report.py]: Done.

Also, I modified the logging information when using `-o` flag.

File: bangumi_report.py [hunk 2]
```diff
@@ -203,40 +198,45 @@ def generate_markdown(self):
...
+            content = self.generate_markdown()
+        
+        if not to_stdout:
+            logging.info('Output file: %s', self.file_name)
+            with open(self.file_name, 'w', encoding='utf-8') as f:
+                f.write(content)
         else:
-            self.generate_markdown()
+            logging.info('No output file, write to stdout directly')
+            print(content)
```
Comment (MichaelMai2000) [inline bangumi_report.py]: Here ;)

**Review by MichaelMai2000 — COMMENTED** [review]
@Shell32-Natsu Is this fix acceptable?

**Pull request merged**